src/main/resources/templates/readPosts.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org" >
```

Comment:
- should lines 2 & 3 be merged so there's only one <html> tag?
- should the Spring Security XML Namespace be referenced as `xmlns:sec=`? and is it actually necessary as it doesn't appear to be referenced in this file?